### PR TITLE
Upgrade azuredisk/azurefile to v1.2.0

### PR DESCRIPTION
**Pull request by ZeroMagic**
Upgrade azuredisk/azurefile to v1.2.0

### CODE REVIEW

File: vhdbuilder/packer/configure-windows-vhd.ps1 [hunk 1]
```diff
@@ -121,7 +121,9 @@ function Get-ContainerImages {
                 "mcr.microsoft.com/oss/kubernetes-csi/azuredisk-csi:v1.0.0",
```
Comment (AbelHu) [inline vhdbuilder/packer/configure-windows-vhd.ps1:121]: Can we remove those old csi versions?

Comment (ZeroMagic) [reply vhdbuilder/packer/configure-windows-vhd.ps1]: Yep. Remove the unnecessary images.

**Review by AbelHu — APPROVED** [review]

**Comment by AbelHu** [comment]
@ZeroMagic You should use a private branch instead of using a cloned repo to submit the request.

**Review by AbelHu — CHANGES_REQUESTED** [review]
You should use a private branch instead of using a cloned repo to submit the request.

**Comment by ZeroMagic** [comment]
Close this PR due to using a cloned repo

**Comment by ZeroMagic** [comment]
Here is the new one https://github.com/Azure/AgentBaker/pull/817.

**Pull request closed without merging**